crates/task-impls/src/upgrade.rs

```diff
@@ -139,17 +125,21 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                 broadcast_event(Arc::new(HotShotEvent::UpgradeVoteSend(vote)), &tx).await;
             }
             HotShotEvent::UpgradeVoteRecv(ref vote) => {
-                debug!("Upgrade vote recv, Main Task {:?}", vote.get_view_number());
+                error!("Upgrade vote recv, Main Task {:?}", vote.get_view_number());
```

Comment:
Does receiving an upgrade vote imply an error?

Reply:
Oh, oops -- good catch! Back to `debug`!